doc/installation.rst

```diff
@@ -59,8 +59,8 @@ dependency can be installed automatically along with PyMongo::
 
   $ python3 -m pip install "pymongo[ocsp]"
 
-Wire protocol compression with snappy requires `python-snappy
-<https://pypi.org/project/python-snappy>`_::
+Wire protocol compression with snappy requires `cramjam
+<https://pypi.org/project/cramjam>`_::
```

Comment:
For backwards compat I think we should to still support python-snappy as a fallback (to avoid breaking snappy users who just upgrade pymongo). Let's say that either can be used but cramjam is preferred if it is installed.

Reply:
Done